plugins/remotemetrics/message.go

```diff
@@ -19,10 +19,6 @@ func onMessageDiscarded(messageID tangle.MessageID) {
 }
 
 func sendMessageSchedulerRecord(messageID tangle.MessageID, recordType string) {
-	if !deps.Tangle.Synced() {
```

Comment:
Can we use bootstrapped here? So the nodes that are bootstrapping won't spam our metrics.

Reply:
Done

Reply:
Is it? I meant still having this check but like the following:
```
	if !deps.Tangle.Bootstrapped() 
```
This way nodes won't spam our metrics before they bootstrap.

Reply:
Already fixed it